packages/flutter/lib/src/animation/animation.dart

```diff
@@ -63,6 +63,8 @@ abstract class Animation<T> extends Listenable implements ValueListenable<T> {
 
   /// Stop calling the listener every time the value of the animation changes.
   ///
+  /// If `listener` is not currently registered as a listener, this is a no-op.
```

Comment:
This ("no-op") is informal language. Prefer something along the lines of "If `listener` is not currently registered as a listener, this method does nothing." or some such. (Informal language is much harder for non-native speakers to understand, especially if it's not in the dictionary.)

Reply:
Done